extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -78,6 +79,7 @@ export default class MainController implements vscode.Disposable {
 
 		this.extensionContext.subscriptions.push(treeView);
 
+		IconPathHelper.setExtensionContext(this.extensionContext);
```

Comment:
This should be one of the first things you do - and definitely done before creating the tree view since that actually uses the icons.

Reply:
Makes sense and changed, though doesn't the view not go live until after `activate()` completes anyway?

Reply:
Yeah in this case you're probably right since the view using this is also contributed by the extension. Still good to do for other purposes though since there are things that use icons that might be loaded immediately.